server/index.js

```diff
@@ -0,0 +1,67 @@
+const { PrismaClient } = require('@prisma/client');
+
+const express = require('express');
+const bodyParser = require('body-parser');
+const bcrypt = require('bcrypt');
```

Comment:
Am curious, which metrics did you use to chose this package?

Reply:
Besides incorporating a salt to protect against rainbow table attacks, bcrypt is an adaptive function: over time, the iteration count can be increased to make it slower, so it remains resistant to brute-force search attacks even with increasing computation power, i found this as a resounding complement to bcrypt from various sources